licenses/library.go

```diff
@@ -117,7 +118,14 @@ func Libraries(ctx context.Context, classifier Classifier, includeTests bool, ig
 			klog.Errorf("Package %s does not have module info. Non go modules projects are no longer supported. For feedback, refer to https://github.com/google/go-licenses/issues/128.", p.PkgPath)
 			return false
 		}
-		licensePath, err := Find(pkgDir, p.Module.Dir, classifier)
+
+		stopAt := p.Module.Dir
+		if isVendorModule(newModule(p.Module), pkgDir) {
+			splits := strings.SplitN(pkgDir, "/vendor/", 2)
+			stopAt = path.Join(splits[0], "vendor", p.Module.Path)
```

Comment:
nit: I would extract this to a method to reuse it, like `vendoredModuleDir(modulePath string) string`